async-nats/src/jetstream/kv/mod.rs

```diff
@@ -1370,14 +1373,25 @@ impl Display for UpdateErrorKind {
         match self {
             Self::InvalidKey => write!(f, "key cannot be empty or start/end with `.`"),
             Self::TimedOut => write!(f, "timed out"),
+            Self::WrongLastRevision => write!(f, "wrong last sequence"),
```

Comment:
You missed the log message change 🙂

Reply:
You are very correct, one moment 😄 

I had a little too much going on this morning and should have re-read the diff

Reply:
There we are